### PR TITLE
Show when no results are found.

### PR DESCRIPTION
**Pull request by jelly**

### CODE REVIEW

File: src/header.tsx [hunk 1]
```diff
@@ -56,6 +56,7 @@ export const FilesCardHeader = ({ currentFilter, onFilterChange, isGrid, setIsGr
                 <SearchInput
                   placeholder={_("Filter directory")} value={currentFilter}
                   onChange={onFilterChange}
+                  onClear={event => onFilterChange(event as React.FormEvent<HTMLInputElement>, "")}
```
Comment (cockpituous) [inline src/header.tsx:59]: This added line is not executed by any test.

Comment (jelly) [reply src/header.tsx:59]: We can, will do.

Comment (jelly) [reply src/header.tsx:59]: Added.

File: src/header.tsx [hunk 2]
```diff
@@ -69,6 +69,7 @@ export const FilesCardHeader = ({
...
```
Comment (cockpituous) [inline src/header.tsx:72]: This added line is not executed by any test.

**Review by martinpitt — APPROVED** [review]
I didn't see the update here. LGTM, thanks!

**Pull request merged**